include/SVFIR/SVFType.h

```diff
@@ -31,6 +31,8 @@
 #ifndef INCLUDE_UTIL_SVFBASICTYPES_H_
 #define INCLUDE_UTIL_SVFBASICTYPES_H_
 
+#include <llvm/Support/Debug.h> // TODO: Remove LLVM Header
```

Comment:
Can you remove this header and move to cpp?

Reply:
Removed.